### DISCUSSION TITLE
有没有类似于NotifyIcon的玩意儿呢（我不太知道怎么描述这个东西）

### DISCUSSION THREAD

**Discussion by clear-zhao** [iNKORE-NET/UI.WPF.Modern#105] [Q&A] [answered]
我在使用Hardcodet.NotifyIcon.Wpf库的时候，在CodeBehind里写气球提示信息的时候发现INKORE好像也有这玩意儿，但是不知道怎么用呢
![image](https://github.com/user-attachments/assets/8192b89c-af1d-4f95-80b7-c8a648ca2402)

**Comment by NotYoojun** [comment, marked as answer]
> 我在使用Hardcodet.NotifyIcon.Wpf库的时候，在CodeBehind里写气球提示信息的时候发现INKORE好像也有这玩意儿，但是不知道怎么用呢
> ![image](https://github.com/user-attachments/assets/8192b89c-af1d-4f95-80b7-c8a648ca2402)
> 

有，在 iNKORE.UI.WPF.TrayIcons 命名空间内，用法和 hardcodet 的差不多，不过 NotifyIcon 要变成 TrayIcon
